stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
```

Comment:
That field was already there and I didn't rock the boat. That is a good point though.